app/models/item.rb

```diff
@@ -1,5 +1,5 @@
 class Item < ActiveRecord::Base
-  validates :name, presence: true
```

Comment:
Request for Change: Did you test this validation?

Reply:
yes, the model specifically in `model/item_spec` and the last test for POST in the controller also provides some test coverage.